clients/scanoss/src/main/kotlin/ScanOssService.kt

```diff
@@ -38,6 +38,11 @@ typealias FullScanResponse = Map<String, List<ScanResponse>>
 
 interface ScanOssService {
     companion object {
+        /**
+         * The default API URL to use.
```

Comment:
Consider stripping "to use".

Reply:
This was moved from `ScanOssConfig.kt`, but as I'm slightly adjusting the formatting anyway, I agree I could as well drop this suffix. Will do.

Reply:
Hadn't notice it was moved...